### ISSUE TITLE
Getting not "not enough space in the buffer"

### ISSUE THREAD

**Issue by amit2103** [PromtEngineer/localGPT#434]
Thanks for creating the awesome project.
So I was trying to play around with the project. 
I have a couple of PDF's that i wanted to use (mostly around 300 pages).

I have a laptop 3080 with 16GB Vram and I was using the , MODEL_ID = "TheBloke/Llama-2-7B-Chat-GGML with MODEL_BASENAME = "llama-2-7b-chat.ggmlv3.q2_K.bin".

I repeatedly get the error 

```ggml_allocr_alloc: not enough space in the buffer (needed 290308608, largest block available 24674304)
GGML_ASSERT: C:\Users\amitp\AppData\Local\Temp\pip-install-tfpb1y6w\llama-cpp-python_c92100fd9be945f4b6d8e996dc576b27\vendor\llama.cpp\ggml-alloc.c:139: !"not enough space in the buffer"```

What is the reason for this, I double checked and the VRAm of the GPU is available (atleast 12GB)

**Comment by aaltulea** [comment]
i get this error too with 4090, 24GB VRAM + 32GB system memory. Windows 11.  Model: TheBloke/Llama-2-7B-Chat-GGML, on: cuda

**Comment by omkar806** [comment]
yes same issue I am having I am using my own appllication to host on streamlit and I am running into the same issue ...so you shifted to vicuna and it works ?
